packages/cli/src/swagger/specGenerator2.ts

```diff
@@ -222,7 +222,7 @@ export class SpecGenerator2 extends SpecGenerator {
 
     const operation: Swagger.Operation = {
       operationId: this.getOperationId(method.name),
-      produces: ['application/json'],
+      produces: [DEFAULT_RESPONSE_MEDIA_TYPE], // todo: Tsoa.Response.produces
```

Comment:
We should fill in the `produces` here, right?

Reply:
> We should fill in the `produces` here, right?

as opposed to `packages/cli/src/swagger/specGenerator3.ts` where there are `buildPaths()` and `buildOperation()` this would require major refactor.

unfortunately I don't want to do it in this PR

Reply:
> We should fill in the `produces` here, right?

indeed, will do